src/bin/bat/clap_app.rs

```diff
@@ -257,6 +257,15 @@ pub fn build_app(interactive_output: bool) -> ClapApp<'static, 'static> {
                      an interactive terminal is detected. Possible values: *auto*, never, always.",
                 ),
         )
+        .arg(
+            Arg::with_name("always-decorations")
+                .short("f")
+                .alias("always-decor")
```

Comment:
I don't see much value in this. For me, long options are useful for scripts, due to the self-documenting character. Short options are useful when I have to type it in the console. An abbreviated long option would be for what?